.github/actions/yaml-parser-action/action.yml

```diff
@@ -0,0 +1,14 @@
+name: "yaml parser"
+author: mattdavis0351
+description: "parse github workflows for info"
+inputs:
+  github-token:
+    description: "Token for API access, use GITHUB_TOKEN"
+    required: true
+  files:
+    description: "Comma separated list of files to read, files must live inside .github/workflows/"
+    required: true
+
+runs:
+  using: "node12"
+  main: "dist/index.js"
```

Comment:
@hectorsector, this MAY need to change to be a more absolute path if we are running it from the local repo, so if for some reason it doesn't run as expected, I would look at this line specifically